RobotStateMessage sourceRobotState
geometry_msgs/Quaternion orientation
geometry_msgs/Vector3 position
---
RobotStateMessage newRobotState